terraform/shared/modules/ecr/ecr.tf

```diff
@@ -11,6 +11,43 @@ resource "aws_ecr_repository" "gdx_data_share_poc" {
   }
 }
 
+resource "aws_ecr_lifecycle_policy" "gdx_data_share_poc" {
```

Comment:
This doesn't seem to do quite what I'd expect. If my latest prod tagged image here is 15 days old I think this will expire it?

Reply:
Missed the untagged specifier